Conference-Android/app/src/main/AndroidManifest.xml

```diff
@@ -47,12 +47,12 @@
         <activity
```

Comment:
This comes down to understanding the up button versus the back button.  The back button is the button that is on the device near the bottom.  (On Nexus, it is not a physical button).  It is designed to take you back the chain that you navigated to get to the screen that you are on.  

The up button is to take you up to another part of the application where you can take a completely different path.  

This change basically made the up button behave the same as the back button.  I don't think that is what we want here.